scripts/release/msigdb-gsea/build_gsea.pl

```diff
@@ -30,13 +30,13 @@
 
 my %repositories = (
     'Pathway-Exchange' => {
-        url => 'https://github.com/reactome/Pathway-Exchange.git',
```

Comment:
Curious: Why remove the ".git", since it looks likeyou will be performing git operations with this URL?

Reply:
I think this was just a holdover from a change I was considering where I used the URL value of the repository.  Since `git clone` works with or without the ".git" suffix, I removed it so the URL was more readable, I suppose, when printed.